.docker/web/etc/supervisor/conf.d/laravel-queue-worker.conf

```diff
@@ -0,0 +1,7 @@
+[program:laravelqueueworker]
+command=php /var/www/ukcp/artisan queue:listen
+numprocs=2
+autostart=true
+autorestart=true
+user=www
+process_name=%(program_name)s_%(process_num)02d

```

Comment:
Is this required if Horizon is responsible for queues?

Reply:
Good spot, no it's not!

Reply:
5f1fcc6